recipes/volk/all/conanfile.py

```diff
@@ -43,6 +43,7 @@ def configure(self):
 
     def requirements(self):
         self.requires("vulkan-headers/" + self.version)
+        self.requires("vulkan-loader/" + self.version)
```

Comment:
You shouldn't do that.

Reply:
But with current behavior, I came to a situation, where vulkan from my system path is used and version does not match. I my case I would use vulkan from conan. What should I do in my case?

Reply:
I don't understand, if you want to use vulkan-loader from conan, add vulkan-loader in requirements of your project. But volk doesn't need vulkan-loader at build time.

Reply:
Yeah, volk should be completely agnostic of the system loader version - if your loader is older, it just means your application wouldn't be able to use some newer extensions, but that's what you would also expect to happen for any user of your software. And you can't use volk together with the statically linked loader in the same application.